### PR TITLE
feat: add model api

### LINKED ISSUE #26: Feature Request: Add /api/model.php query
Currently, there's a query for current keyboard download links
e.g. https://downloads.keyman.com/api/keyboard/sil_euro_latin

It would be nice to have one for models
e.g. https://downloads.keyman.com/api/model/nrc.en.mtnt

Also, should .htaccess be updated so block /api from being browsable?

### PR DESCRIPTION
**Pull request by mcdurdin**
Adds /api/model with a paramset that matches /api/keyboard.

Also tidies up documentation and /api/ root

Fixes #26.

### CODE REVIEW

File: api/model.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    Gets the available download files for a given model id
+  */
+
+  header('Content-Type: application/json; charset=utf-8');
+  header('Cache-Control: max-age=0');
+
+  $allowed_platforms = array('android', 'ios' /*, 'linux', 'mac', 'web', 'windows'*/); // models only currently supported on iOS and Android
+  $allowed_versions = array('1.0'); // api versions, not product versions
+  $release_tiers = array('alpha', 'beta', 'stable');
```
Comment (darcywong00) [inline api/model.php]: Are `$allowed_platforms` and `$release_tiers` for future use? I see they came from keyboard.php

**Review by darcywong00 — APPROVED** [review]
lgtm.

Will this handle multiple id's in the api/model or api/keyboard/ query?

**Comment by mcdurdin** [comment]
> Will this handle multiple id's in the api/model or api/keyboard/ query?

This will be handled in api.keyman.com; we need to remember that this API is for internal web site use, not app use.

**Comment by darcywong00** [comment]
I think from conversations we can add this. But not have the mobile apps use these API's.

**Pull request merged**